.github/workflows/coverage.yml

```diff
@@ -47,6 +47,8 @@ jobs:
           default: true
       - name: Setup cmake
         uses: jwlawson/actions-setup-cmake@v1.12
+      - name: Install Protoc
```

Comment:
same here

Reply:
coverage work with `runs-on: ubuntu-latest` that needs the Protoc

Reply:
no it's self-hosted